tests/utils/utils.go

```diff
@@ -55,15 +64,42 @@ type FeeInfo struct {
 	Amount          *big.Int
 }
 
-func httpToWebsocketURI(uri string, blockchainID string) string {
+//
+// Test utility functions
+//
+
+// Subscribes to new heads, sends a tx, and waits for the new head to appear
+// Returns the new head
+func SendAndWaitForTransaction(
```

Comment:
Nit: Could we renaming this something like `SendTransactionAndWaitForAcceptance`?